Automattic/wp-calypso
# Stats: Improvements to video stats

#### Changes proposed in this Pull Request

This PR updates video stats to add watch time and impressions.
It also changes "plays" to "views" and fixes some existing issues with Video stats.

#### Testing instructions

* On a sanboxed public-api
* Apply D79611-code
* Apply this PR
* Open your Calypso backend
* Select the "Stats" menu
* ✅ The "video stats" section should work.
* Click on the video stats
* ✅ You should see a list of video stats for the selected period
* ✅ You should be able to click on "Download CSV" and generate a CSV from the visible stats
* On a stat line, click on the video title
* ✅ You should be redirected to the media item in the Media library
* Go back
* Click on each stats (views, impressions and hours watched)
* ✅ Each stat should redirect to the appropriate stat section (with some bars)
* ✅ The stats should match
* ✅ The stats should reflect your currently selected time period
* ✅ Using the "year" time period, the graph should display month-by-month
* ✅ Using another time perios, the graph should be daily

Related to Automattic/greenhouse#1170
